Lib/test/test_pkgutil.py

```diff
@@ -489,6 +489,61 @@ def test_mixed_namespace(self):
     # XXX: test .pkg files
 
 
+class ExtendPathBaseTests(unittest.TestCase):
+    def test_input_type_invalid(self):
+        # path should be list instead of string
+        path = 'path'
+        name = 'foo'
+        self.assertEqual('path', pkgutil.extend_path(path, name))
+
+    def test_parent_package_raise_key_error(self):
```

Comment:
```suggestion
    def test_parent_package_raises_key_error(self):
```

Reply:
There's nothing here making sure `KeyError` is raised, so the name of the test is confusing.